### DISCUSSION TITLE
Things to know

### DISCUSSION THREAD

**Discussion by ttugdsc** [ttugdsc/ttugdsc#2] [Announcements]
🛠️ 👩‍💻 **Resources for Developing the Website** ⌨️ 🎨 
- Read up on the [JamStack](https://jamstack.org/) [quick video](https://www.youtube.com/watch?v=ySJGjo3_EX4&t=2s)
Ideally after watching the video you'll have an idea of what we want to achieve by the end of this - a fast website to show who we are with a "blog" of upcoming and past events
- Learn about [static sites and more](https://www.youtube.com/watch?v=Kg0Q_YaQ3Gk)
- Learn about React [in a few minutes](https://www.youtube.com/watch?v=Tn6-PIqc4UM)
- Next.js - A framework for React. [Next.js in 100 seconds or more](https://www.youtube.com/watch?v=nhBVL41-_Cw) explaining what static site generation is
- Learn about git workflows, specifically [Git Flow - video](https://www.youtube.com/watch?v=aJnFGMclhU8)
The good news is, most code editors have git integration along with GitHub having that functionality in the website itself so you don't *need* to learn the commands precisely but its good to know [Crash Course](https://www.youtube.com/watch?v=SWYqp7iY_Tc)

**Comment by Guysnacho** [comment]
So unfortunately since Gridsome has been inactive and there's no timeline for when support for Vue 3 will be live I'm making the decision to build the website with Nuxt.js. Its another framework that has the functionality we want. The same concepts we were shooting for apply, we'll just be building on the latest that Vue and Nuxt have to offer. If you want, you can look to contribute to the Gridsome project. I know I will

**Reply by Guysnacho** [reply]
**Ploooooot twist**, so is turns out that Nuxt is actually overkill for what we're trying to do. To the point that we need to jump through hoops just trying to add custom modules and third-party components. So I'll be migrating us back to Gridsome in the navbar branch. If you haven't already started working yet then I recommend holding off until the next pull request.

tl;dr - The latest and greatest might be a bit much, rolling back to Gridsome. **Wait for the next PR to start working**

**Comment by Guysnacho** [comment]
![](https://cdn.discordapp.com/attachments/773050562069004319/874094070597820497/DSC_Wireframe.png)

Here are our finished wireframes for the site.
